### ISSUE TITLE
  ValueError: too many values to unpack

### ISSUE THREAD

**Issue by hovo1990** [choderalab/pymbar#175]
Trying python alchemical_analysis.py -d gromacs/data/3-methylindole-38steps -q xvg -p dhdl -u kJ -v
and I get

/usr/lib/python2.7/dist-packages/scipy/optimize/minpack.py:236: RuntimeWarning: xtol=0.000000 is too small, no further improvement in the approximate
solution is possible.
warnings.warn(msg, RuntimeWarning)
Final dimensionless free energies
f_k =
[ 0. 2.85435075 5.36683423 7.54310132 9.41062672
11.00680904 12.36821823 13.52539 14.50082038 15.30933876
15.95876113 13.96406986 13.58387343 13.99782737 14.35144787
14.74395871 15.13605675 15.48968283 15.76312185 15.86488891
15.89546715 15.88720731 15.82995611 15.71117856 15.51663687
15.2322169 14.84601322 14.35037166 13.74694486 13.05530028
12.31496947 11.57396348 10.87411229 10.24438184 9.70162777
8.90365663 8.48915993 8.43014626]
MBAR initialization complete.
Traceback (most recent call last):
File "alchemical_analysis.py", line 1103, in
Deltaf_ij, dDeltaf_ij = estimatewithMBAR(u_kln, N_k, P.relative_tolerance, regular_estimate=True)
File "alchemical_analysis.py", line 239, in estimatewithMBAR
(Deltaf_ij, dDeltaf_ij) = MBAR.getFreeEnergyDifferences(uncertainty_method='svd-ew')
ValueError: too many values to unpack

**Comment by hovo1990** [comment]
Line 239 in alchemical_analysis  has to be changed to:

 (Deltaf_ij, dDeltaf_ij,Theta_ij) = MBAR.getFreeEnergyDifferences(uncertainty_method='svd-ew')

**Comment by jchodera** [comment]
Thanks for pointing this out!

@mrshirts : Can you take a look at this?

**Comment by kyleabeauchamp** [comment]
This is fixed in #176, I will merge that PR soon.

**Comment by mrshirts** [comment]
In general, we shouldn't expect people to have to accept the theta coming out of MBAR, since it's never used (and usually won't be used).

**Comment by jchodera** [comment]
I think we discussed this before and made the decision to adopt this form
to simplify the code based on the assumption that this wouldn't break
anything. Unfortunately, the experiment I conducted was incorrect, and
users do need to capture this argument.

If we want to change the behavior, it is back to using optional arguments
to specify which return values are needed, or returning objects or dicts.
